parts/linux/cloud-init/artifacts/disk_queue.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Set nr_requests and queue_depth
+
+[Service]
+Type=oneshot
+ExecStart=bash -c 'echo 128 > /sys/block/sda/queue/nr_requests && echo 128 > /sys/block/sda/device/queue_depth'
```

Comment:
Just curious: is the goal of the PR to be able to change the default value of nr_requests and queue_depth?

Reply:
it's actually just to hardcode it to a new default